### PR TITLE
Remove a inline event handlers

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Remove a lot of inline event handlers as part of the ongoing Content-Security-Policy work.

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Review by labkey-tchad — APPROVED** [review]

File: modules/miniassay/resources/assay/simple/views/upload.html [hunk 1]
```diff
@@ -55,11 +55,10 @@
 <br/>
 <div id='runs_div'></div>
 
-<script type="text/javascript" nonce="<%=getScriptNonce()%>">
-  LABKEY.requiresScript("FileUploadField.js");
+<script type="application/javascript" nonce="<%=scriptNonce%>">
   document.getElementById("X-LABKEY-CSRF").value = LABKEY.CSRF;
 </script>
-<script type="text/javascript" nonce="<%=getScriptNonce()%>">
```
Comment (labkey-tchad) [inline modules/miniassay/resources/assay/simple/views/upload.html]: Why is this changed to be `applications/javascript`? Most other scripts use `text/javascript`, which seems to be the preferred option from a quick search.

Comment (labkey-matthewb) [reply modules/miniassay/resources/assay/simple/views/upload.html]: fixed
